plugins/org.jkiss.dbeaver.ext.mssql/src/org/jkiss/dbeaver/ext/mssql/model/SQLServerSchema.java

```diff
@@ -523,23 +529,35 @@ protected SQLServerTableIndex fetchObject(JDBCSession session, SQLServerSchema o
         {
             int indexTypeNum = JDBCUtils.safeGetInt(dbResult, "type");
             DBSIndexType indexType;
+            // Type of index:
+            // 0 = Heap
+            // 1 = Clustered rowstore (B-tree)
+            // 2 = Nonclustered rowstore (B-tree)
+            // 3 = XML
+            // 4 = Spatial
+            // 5 = Clustered columnstore index. Applies to: SQL Server 2014 (12.x) and later.
+            // 6 = Nonclustered columnstore index. Applies to: SQL Server 2012 (11.x) and later.
+            // 7 = Nonclustered hash index. Applies to: SQL Server 2014 (12.x) and later.
```

Comment:
This comment is now redundant, please remove it

Reply:
No. It is not.

Reply:
It is

Reply:
This comment is useful if we assign each piece of comment to the corresponding constant.